resources/service.rb

```diff
@@ -66,27 +66,19 @@
 end
 
 action :load do
-  unless loaded
-    execute "hab sup load #{service_name} #{sup_options.join(' ')}"
-  end
+  execute "hab sup load #{service_name} #{sup_options.join(' ')}" unless loaded
 end
 
 action :unload do
-  if loaded
-    execute "hab sup unload #{service_name} #{sup_options.join}"
-  end
+  execute "hab sup unload #{service_name} #{sup_options.join}" if loaded
 end
 
 action :start do
-  unless running
-    execute "hab sup start #{service_name} #{sup_options.join}"
-  end
+  execute "hab sup start #{service_name} #{sup_options.join}" unless running
```

Comment:
Same for `running` as `loaded`. There is an empty definition for `service_running?`, but not `running`.